metrics/mixin/rules/rules.libsonnet

```
@@ -1,4 +1,33 @@
 {
   prometheusRules+:: {
+    groups+: [
+      {
+        name: 'ocs_telemetry.rules',
+        rules: [
+          {
+            record: 'ocs:node_num_cpu:sum',
+            expr: |||
+              sum(
+                max(
+                  (
+                    kube_pod_container_resource_requests_cpu_cores{namespace="openshift-storage", pod =~"(noobaa-core.*|noobaa-db.*|rook-ceph-(mon|osd|mgr|mds|rgw).*)"} * 0 +1 
+                  ) * on(node) group_left() node:node_num_cpu:sum
+                ) by (node)
+              )
+            ||| % $._config,
+          },
+          {
+            record: 'ocs_pod:container_cpu_usage:sum',
```

```suggestion
            record: 'ocs:container_cpu_usage:sum',
```
I don't think we need to append `_pod` since we don't group it by `pod` ?

---

Makes sense. Will update.